Logging/Logging/BrainstormSessions/Controllers/SessionController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Threading.Tasks;
+using BrainstormSessions.Core.Interfaces;
+using BrainstormSessions.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace BrainstormSessions.Controllers
+{
+    public class SessionController : Controller
+    {
+        private readonly IBrainstormSessionRepository _sessionRepository;
+        private readonly ILogger<SessionController> _logger;
+
+        public SessionController(
+            IBrainstormSessionRepository sessionRepository,
+            ILogger<SessionController> logger)
+        {
+            _sessionRepository = sessionRepository;
+            _logger = logger;
+        }
+
+        public async Task<IActionResult> Index(int? id)
+        {
+            if (!id.HasValue)
+            {
+                _logger.LogError("SESSION - Session Id is not provided in SessionController");
```

Comment:
You can do it like: 
```
_logger.LogError($"SESSION - Session Id is not provided in {nameof(SessionController)});
```

In this scenario, your real name of controller will be automatically generated with reflection help and you don't have to update message in case of class change

You may also follow this template: `$"{ClassName}."{MethodName}: {YourAwesomeMessage}"`